business_requirement_deliverable_project_categ/wizard/generate_projects.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+
+
+class BrGenerateProjectsWithCategories(models.TransientModel):
+    _inherit = 'br.generate.projects'
+
+    @api.multi
+    def _prepare_project_task(self, line, project_id):
+        vals = super(BrGenerateProjectsWithCategories, self) \
+            ._prepare_project_task(line, project_id)
+        vals.update({'categ_ids': [(6, 0, [line.task_categ_id.id])]})
```

Comment:
@moylop260 Sounds that only we are updating a `dict` but what we pursue is modularity and low impact dependency.

By the way @elicoidal as soon is merged https://github.com/OCA/project/pull/239 I would propose a new module
business_requirement_delivarable_main_categ 
and
business_requirement_delivarable_project_main_categ